libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -556,19 +553,25 @@ static CK_RV prvMbedTLS_Initialize( void )
         mbedtls_entropy_init( &xP11Context.xMbedEntropyContext );
         mbedtls_ctr_drbg_init( &xP11Context.xMbedDrbgCtx );
 
-        if( 0 != mbedtls_ctr_drbg_seed( &xP11Context.xMbedDrbgCtx,
-                                        mbedtls_entropy_func,
-                                        &xP11Context.xMbedEntropyContext,
-                                        NULL,
-                                        0 ) )
+        lMbedTLSResult = mbedtls_ctr_drbg_seed( &xP11Context.xMbedDrbgCtx,
+                                                mbedtls_entropy_func,
+                                                &xP11Context.xMbedEntropyContext,
+                                                NULL,
+                                                0 );
+
+        if( 0 != lMbedTLSResult )
         {
+            LogError( ( "Failed to seed the mbed TLS DRBG for the PKCS #11 module with error %s",
+                        mbedtlsHighLevelCodeOrDefault( lMbedTLSResult ) ) );
+            LogError( ( "%s", mbedtlsLowLevelCodeOrDefault( lMbedTLSResult ) ) );
```

Comment:
The high and low level code strings can be logged in the same call
```suggestion
            LogError( ( "Failed to seed the mbed TLS DRBG for the PKCS #11 module: mbedTLSError=%s : %s",
                        mbedtlsHighLevelCodeOrDefault( lMbedTLSResult ),
                        mbedtlsLowLevelCodeOrDefault( lMbedTLSResult ) ) );
```

Reply:
Same for other logging sites of mbedTLS error strings